docs/workflow.md

```diff
@@ -84,10 +84,9 @@ If you do rebase or merge and get conflicts, you'll need to resolve them manuall
 ### Formatting Your Code
 We use [clang-format](https://electronjs.org/docs/development/clang-format) to automatically format our code. Using an automatic tool helps keep things consistent across the codebase without developers having to change their personal style as they write. See the [code style guide](code-style-guide.md) for more information on exactly what it does.
 
-To format the code, from the `Software` directory run `./clang_format/fix_formatting.sh -b master`. This will take the difference between your current branch and your local `master` branch, and run `clang-format` on those changes. The only reason we use the `-b master` option is because it's faster and only formats code you have modified. There is also a `-a` option that will try format the entire codebase, but it is slow and not recommended.
-
-We recommend committing all your changes and then running the formatting script. This keeps the formatting changes separate from your actual changes. For the formatting changes, a simple commit message like "fixed formatting" is fine.
+To format the code, from the `Software` directory run `./clang_format/fix_formatting.sh -b upstream/master`. This will take the difference between your current branch and the `upstream/master` branch, and run `clang-format` on those changes. The only reason we use the `-b upstream/master` option is because it's faster and only formats code you have modified. There is also a `-a` option that will try format the entire codebase, but it is slow and not recommended.
```

Comment:
I missed updating this when I merged the new formatting scripts, but it should just be:
``` suggestion
To format the code, from the `Software` directory run `./formatting_scripts/fix_formatting.sh`.
```
As we're formatting the entire codebase now to prevent weird formatting issues that kept popping up in PR's.

Reply:
Ah yes that was the original reason why I looked at that part of documentation but I forgot to change it